src/components/ebay-star-rating/README.md

```diff
@@ -14,12 +14,15 @@
 
 ```marko
 <ebay-star-rating-1/>
```

Comment:
so we’ll be phasing out the old syntax <ebay-star-rating-1/> or not?

Reply:
Yes we have both
A so its not breaking 
B so its in line with skin

I think the new one is dynamic and the other one is fixed or static in storybook